lib/extensions.js

```diff
@@ -66,6 +66,12 @@ module.exports = function(realmConstructor) {
 
         setConstructorOnPrototype(realmConstructor.Sync.User);
         setConstructorOnPrototype(realmConstructor.Sync.Session);
+    } else {
+        Object.defineProperty(realmConstructor, 'Sync', {
+            get: function () {
+                throw new Error("Sync is not enabled. Note that the developer edition of the Node.JS SDK for Realm does not include sync on Linux.");
```

Comment:
Maybe wording could be improved. Is this "not enabled" or it is "not available". Not sure about that.